test/parallel/test-path-parse-format.js

```diff
@@ -30,13 +30,13 @@ var unixPaths = [
 ];
 
 var errors = [
-  {method: 'parse', input: [null], message: /Parameter 'pathString' must be a string, not/},
+  {method: 'parse', input: [null], message: /Parameter 'pathString' must be a string, not null/},
```

Comment:
Keep in mind these are test case regexes, not descriptions.

This case fails because the returned string is actually `'Parameter 'pathString' must be a string, not object'` due to type coercion.